ompl missing dependency

After installing the dependencies with rosdep and wstool, seems like a missing dependency. These issues are found when trying to build in an isolated (docker) environment.

```
Errors     << terrain_planner:cmake /root/catkin_ws/logs/terrain_planner/build.cmake.000.log                                                                                                                                              
CMake Error at /root/catkin_ws/src/terrain-navigation/terrain_planner/CMakeLists.txt:10 (find_package):
  By not providing "Findompl.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "ompl", but
  CMake did not find one.

  Could not find a package configuration file provided by "ompl" with any of
  the following names:

    omplConfig.cmake
    ompl-config.cmake

  Add the installation prefix of "ompl" to CMAKE_PREFIX_PATH or set
  "ompl_DIR" to a directory containing one of the above files.  If "ompl"
  provides a separate development package or SDK, be sure it has been
  installed.
  ```
  
  https://github.com/ros-gbp/ompl-release
  
  Please assign this to me, I'll issue a fix. If you have a custom fork of ompl rather than binaries, let me know.

--- a/terrain_navigation_ros/package.xml
+++ b/terrain_navigation_ros/package.xml
@@ -34,6 +34,7 @@
   <build_depend>terrain_navigation</build_depend>
   <build_depend>terrain_planner</build_depend>
   <build_depend>dynamic_reconfigure</build_depend>
+  <build_depend>ompl</build_depend>
   <run_depend>roscpp</run_depend>
   <run_depend>rospy</run_depend>
   <run_depend>grid_map_visualization</run_depend>
@@ -43,6 +44,7 @@
   <run_depend>terrain_planner</run_depend>
   <run_depend>dynamic_reconfigure</run_depend>
   <run_depend>mav_planning_rviz</run_depend>
+  <run_depend>ompl</run_depend>
   <export>
   </export>
 </package>

--- a/terrain_planner/package.xml
+++ b/terrain_planner/package.xml
@@ -24,6 +24,7 @@
   <build_depend>tf</build_depend>
   <build_depend>planner_msgs</build_depend>
   <build_depend>terrain_navigation</build_depend>
+  <build_depend>ompl</build_depend>
   <run_depend>roscpp</run_depend>
   <run_depend>rospy</run_depend>
   <run_depend>grid_map_visualization</run_depend>
@@ -31,6 +32,7 @@
   <run_depend>planner_msgs</run_depend>
   <run_depend>terrain_navigation</run_depend>
   <run_depend>grid_map_geo</run_depend>
+  <run_depend>ompl</run_depend>
 
 
   <!-- The export tag contains other, unspecified, tags -->

--- a/terrain_planner_benchmark/package.xml
+++ b/terrain_planner_benchmark/package.xml
@@ -22,6 +22,7 @@
   <build_depend>grid_map_pcl</build_depend>
   <build_depend>grid_map_geo</build_depend>
   <build_depend>terrain_navigation</build_depend>
+  <build_depend>ompl</build_depend>
   <build_depend>terrain_planner</build_depend>
   <run_depend>eigen_catkin</run_depend>
   <export>

